### ISSUE TITLE
update to 2.2.0 breaks stuff

### ISSUE THREAD

**Issue by mslucasMMID** [telerik/kendo-icons#238]
Hi,

Since updating from 2.1.0 to 2.2.0 some things don't work anymore.
I use
Telerik.FontIcons.FontIcon.Plus
Telerik.FontIcons.FontIcon.FileExcel

But they don't work now

**Comment by Juveniel** [comment]
Hey @mslucasMMID,

Could you provide the setup you are working with so that we can investigate further ?

**Comment by mslucasMMID** [comment]
I use it like this in a Blazor application

```
@using Telerik.FontIcons

<TelerikGrid Data="@GridData">
    <GridToolBarTemplate>
        <GridCommandButton Command="ExcelExport" Icon="@FontIcon.FileExcel">Export to Excel</GridCommandButton>
        <label class="k-checkbox-label"><TelerikCheckBox @bind-Value="@ExportAllPages"/>Export All Pages</label>
    </GridToolBarTemplate>
....

```

**Comment by radkostanev** [comment]
Hey, @mslucasMMID,

Thanks for bring this up - we will look into it.

As for your use-case - if you are using the Telerik UI for Blazor components, then you should not have to manually reference the icon NuGets. The [way the icons in Telerik UI for Blazor work](https://docs.telerik.com/blazor-ui/common-features/icons#how-icons-work) is that they are automatically installed, as they are a dependency to the component package. In other words, you can omit the package reference declarations within your csproj file - the icons will be correctly resolved and the latest working version will be used (that is `2.1.0`). Telerik UI for Blazor has not yet released a newer version using the `2.2.0` version of the `Telerik.SvgIcons`, thus this should not affect the UI components.

If I am missing something, could you please clarify what causes the need to reference them separately?

**Comment by mslucasMMID** [comment]
I removed the Telerik.FontIcons package and indead he now uses the 2.1.0 indirectly and everything works.

No idea why I installed the package by hand in the past.
